packages/core/src/api/services.ts

```diff
@@ -83,7 +84,9 @@ export function composeContentNodeOptions(
 }
 
 export function parseContentNodeQuery(
-  result: ApolloQueryResult<WPGraphQL.GetContentNodeQuery>,
+  result:
+    | ApolloQueryResult<WPGraphQL.GetContentNodeQuery>
+    | QueryResult<WPGraphQL.GetContentNodeQuery>,
```

Comment:
`QueryResult` was added back as this is what Apollo's [`useQuery`](https://github.com/wpengine/headless-framework/pull/217/files#diff-c6ce8673cb12abb4ff3134573c3d9a414e9063997fbc8cddf6912cd8bcac3ff5R61) still returns.